html_css/project/html-css-foundations-project.md

```diff
@@ -0,0 +1,81 @@
+
+Do _not_ be afraid to use google or go back to previous lessons to look something up. **Real-life, professional developers use google _constantly_ for things that they have been doing for years.** At this point it is not expected that you will have everything memorized so don't worry about it. Additionally, there are a few small details that you may not have encountered in our lessons yet. _This is by design._ These details are minor, and easily searched (i.e. google `css rounded corners`).
+
+Get your project as close as you can to the design but do not worry about getting it pixel-perfect. Don't get out your ruler or count pixels to find the exact margin between the various sections. The point of this assignment is creating something from scratch and getting the various elements in more or less the right position relative to the rest. It doesn't matter if you use `margin: 24px` when the design actually has `margin: 48px`.
+
+_Finally_, feel free to substitute your own content into this design. The images have some meaningless dummy-content, but if you want to make up a business and personalize this page, please feel free to do so! Insert actual images in the placeholders, and feel free to play with the colors and fonts a bit too.
+
+### Setting up your project's GitHub repository
+
+As mentioned in the [introduction to Git](/courses/foundations/lessons/introduction-to-git), you'll want to organize all your projects like a portfolio and link them to GitHub so it can be seen by others.
+
+If you do not know how to set up a repository, follow the instructions found in [Git Basics](/courses/foundations/lessons/git-basics) to learn how.
+
+1. If you haven't already, create a folder on your computer called `the_odin_project` and `cd` into it. This folder will house all the projects you do at Odin.
+2. Create a new repo for this project on GitHub.com and call it `foundations-project` (instead of `git-test`).
+3. Then move that repository onto your local machine. The command should look like: `git clone git@github.com:username/foundations-project.git` (use SSH).
+
+   - [Setting up SSH on GitHub](/courses/foundations/lessons/setting-up-git#step-2-configure-git-and-github).
+
+4. Now `cd` into the `foundations-project` directory that is now on your local machine; set up your `README.md` file and write a brief introduction for what the current project is and what skills you have demonstrated once you have completed it. (You can do this as a self-reflection at the end of the project which is a good way to review what you have learned.)
+
+If you are having trouble:
+
+- Ensure you followed the steps [here on Step 2.3](/courses/foundations/lessons/setting-up-git#step-2-configure-git-and-github) to clone from GitHub with SSH.
+- Refer to the [workflow](/courses/foundations/lessons/git-basics) in the Git Basics Lesson.
+
+Note: All Git commands need to be run from inside your project's folder (did you forget to `cd` into the `foundations-project` folder?)
```

Comment:
I think the "Note: All Git commands...." line coudl be added as a third bullet point. "Make sure all of your Git commands are being ran inside your project's folder (did you forget to `cd` into the `foundations-project` folder?).

For the first bullet, the verbiage around the link makes it sound like it is directing to how to clone a repo from Github onto our machines, but the link and step mentioned is about setting up SSH itself. Not sure if the link should be for the Git Basics lesson on cloning from GitHub or if the verbiage should instead be "Ensure you followed the steps...to create an SSH key in order to clone from GitHub using SSH.